modules/Microsoft.EventHub/namespaces/.test/pe.parameters.json

```diff
@@ -0,0 +1,22 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+        "name": {
+            "value": "<<namePrefix>>-az-evhns-pe-001"
+        },
+        "privateEndpoints": {
+            "value": [
+                {
+                    "subnetResourceId": "/subscriptions/<<subscriptionId>>/resourceGroups/validation-rg/providers/Microsoft.Network/virtualNetworks/adp-<<namePrefix>>-az-vnet-x-001/subnets/<<namePrefix>>-az-subnet-x-005-privateEndpoints",
```

Comment:
regarding `subnetResourceId`, now we will have multiple variations. One with `subnetId` and this version. So this means we are making it easier on one type, and not on other. How would that look from a user experience?

Reply:
I'd vote for consistency and using "resourceId"

Reply:
This means we no longer need to create the "variable" in the bicep file that handles this right? So we should revert to the original model?

Reply:
I'm confused, what is the suggestion here?

Reply:
So now in some places we have subnetId because we simplified the original property for a better user experience. But in another section like for private endpoints, it's subnetResourceId, because we stuck to the original way of how it should be referenced. If this isn't confusing for anyone except me, then all good 😀

Reply:
So the proposal would simply be to rename `subnetId` to `subnetResourceId` for consistency? Would that solve the confusion? I got a bit lost with the proposal of reverting the work done on the variable.

Reply:
Sorry my bad I'd still favour keeping things as they align to the Reference documentation hence the suggestion. But let's go with subnetResourceId. At least not too different.

Reply:
Updated both eventhub and servicebus to `subnetResourceId` and retested